.github/workflows/test-web-app.yml

```diff
@@ -18,6 +18,8 @@ env:
   NEXT_PUBLIC_NETWORK: "Preprod"
   ADMIN_KEY: ${{ secrets.ADMIN_KEY }}
   REGISTRY_API_KEY: ${{ secrets.REGISTRY_API_KEY }}
+  PAYMENT_API_KEY: ${{ secrets.PAYMENT_API_KEY }}
```

Comment:
That doesn't work for external contributor pull requests and could leak information.
We have to use the default secret variable.

Reply:
Should be solved in https://github.com/masumi-network/sokosumi/pull/208/commits/2f6bdf5f138f3066fd105c8eb76dafd660a68b56